crates/signal/src/lib.rs

```diff
@@ -65,15 +63,128 @@
 use std::pin::Pin;
 use std::task::{Context, Poll};
 
-use tokio::signal::unix::{Signal, SignalKind};
+#[cfg(unix)]
+use tokio::signal::unix::{Signal as UnixSignal, SignalKind as UnixSignalKind};
```

Comment:
we should just import this as unix to be the same with windows; and then we should reexport the `unix::UnixSignalKind`